kernel-kit/6.1.x-x86-build.conf

```diff
@@ -0,0 +1,129 @@
+#--
+#             Configuration options
+#             =====================
+# 
+#  **NOTE**: check the original file every once in a while
+#            settings might be added or removed...
+#
+# see http://www.kernel.org/ for the latest kernel info..
+#--
+
+################################################################
+## add a Kernel .config file in one of the configs_* directories
+## It must follow this syntax:
+##           version  info
+## DOTconfig-3.14.73-i686-4g
+################################################################
+
+## speed up the process by specifing a DOTconfig file:
+#DOTconfig_file=DOTconfig
+DOTconfig_file=configs_x86/DOTconfig-6.1.52-x86
+
+# latest in series, LatestK 
+# this gets the latest kernel in relation to the DOTconfig_file
+# specified above and depends on that value. It must be a longterm
+# kernel
+LatestK=1
+
+## use this version string with the current ./DOTconfig
+## (in case it doesn't have usable version info inside)
+kernel_ver=
+
+## i386 specific stuff: force pae/nopae - i486/i686
+## note: the script will run 'make oldconfig' to ensure
+##       the new settings take effect..
+#x86_disable_pae=yes
+#x86_enable_pae=yes
+
+## this is the kernel suffix hack, you can leave it empty ie: uname -r
+## if you don't leave empty the leading "dash" is required
+## DO NOT change it between minor versions (i.e 3.14.1 and 3.14.2); otherwise,
+## third-party drivers will break and users won't be able to upgrade: kernel-kit
+## patches the kernel to ensure drivers built for the major version (say, 3.14)
+## continue to work with any future minor version (3.14.1, 3.14.2, etc') and
+## don't have to be rebuilt, but the suffix must not change for this to work
+## (this is particularly good if you wish to stick with a longterm kernel branch
+## without any maintenance and without frustrating users that need extra drivers)
+## example:
+#custom_suffix=-4G
+custom_suffix=
+
+## this is the name of the pet package suffix, and source sfs
+## name it whatever you like, usually put in a signifier for your distro
+## eg: "s" is for slacko
+## if you leave empty the script will determine a proper package suffix
+package_name_suffix=kernel-kit
+
+##-----------------------------------------------------------------------
+
+## remove kernel sublevel, or not : set yes or no
+remove_sublevel=no
+
+## aufs version (git branch) - see README -> AUFS GIT BRANCHES
+## the script automatically detects the aufs version, but when it does not
+## you must specify it here:
+#aufsv=3.14.40+
+
+## JOBS ###
+## if you have a multicore processor you can set this var
+## don't set if you have a single core! >> cooked machine 
+## DO NOT set it to 0 (zero) >> cooked machine
+#JOBS=-j6
+
+### squashfs compression ###
+## unset this for the default of your mksquashfs binary
+#COMP="-comp gzip"
+#COMP="-comp xz"
+COMP="-comp xz -b 512K"
```

Comment:
Some users (netbooks without much L1/2/3 cache?) say they can't use a Puppy that uses xz with block size larger than 256K